### ISSUE TITLE
Null Pointer Exception

### ISSUE THREAD

**Issue by SevereOverfl0w** [oakes/Nightlight#21]
```
                              java.lang.Thread.run              Thread.java:  745
java.util.concurrent.ThreadPoolExecutor$Worker.run  ThreadPoolExecutor.java:  617
 java.util.concurrent.ThreadPoolExecutor.runWorker  ThreadPoolExecutor.java: 1142
               java.util.concurrent.FutureTask.run          FutureTask.java:  266
                                               ...                               
               clojure.core/binding-conveyor-fn/fn                 core.clj: 2020
                                 boot.core/boot/fn                 core.clj: 1029
                               boot.core/run-tasks                 core.clj: 1019
                 nightlight.boot/eval1963/fn/fn/fn                 boot.clj:   14
                             nightlight.core/start                 core.clj:  158
                             nightlight.core/start                 core.clj:  163
                     org.httpkit.server/run-server               server.clj:    8
                                               ...                               
java.lang.NullPointerException: 
    clojure.lang.ExceptionInfo: 
    file: "/tmp/boot.user2374817428930059651.clj"
    line: 63

```

Using boot version 2.7.1.

**Comment by SevereOverfl0w** [comment]
Looks like this happens when you don't specify a port

**Comment by oakes** [comment]
Fixed in 725d548a31a4714523c38fceb14c785bb50936d6, thanks!
